src/core/dom/event-helper-listen.js

```diff
@@ -92,10 +92,23 @@ export function detectEvtListenerOptsSupport() {
     const options = {
       get capture() {
         optsSupported = true;
+        return false;
       },
     };
-    self.addEventListener('test-options', null, options);
-    self.removeEventListener('test-options', null, options);
+    self.addEventListener(
```

Comment:
It probably would, but a) that adds a few bytes and b) I don't know for certain. Wherever possible in this PR, I erred on the side of type annotations over code changes since they have no bundle size or functionality impact (even though they're super ugly).